api/config/config_perf.json

```diff
@@ -42,6 +42,7 @@
     ]
   },
   "auth": {
+    "extractionServiceAccount": "wgs-cohort-extraction@all-of-us-rw-perf.iam.gserviceaccount.com",
```

Comment:
I think naming should be more specific: `genomicsExtractionServiceAccount`? `cohortExtractionServiceAccount` ? `cromwellServiceAccount` ?